conf/rubocop.yml

```diff
@@ -18,6 +18,7 @@ Metrics/BlockLength:
   - "spec/**/*.rb"
   - "lib/tasks/**/*.rake"
   - "config/environments/*.rb"
+  - "app/graphql/**/*.rb"
```

Comment:
I wonder if we'd want to put this in the [rails specific conf](https://github.com/ezcater/ezcater_rubocop/blob/master/conf/rubocop_rails.yml)?

Reply:
Unrelated to this change, but the `config/environments/*.rb` and `lib/tasks/**.*rake` exclusions should also probably be moved to the Rails-specific config file.